profile/Profile.proto

```diff
@@ -40,3 +50,32 @@ message ProfileTaskCommandQuery {
     // last commmand timestamp
     int64 lastCommandTime = 3;
 }
+
+// dumped segment snapshot
+message ProfileTaskSegemntSnapshot {
+    // profile task id
+    string taskId = 1;
+    // dumped segment id
+    UniqueId traceSegmentId = 2;
+    // dump timestamp
+    int64 time = 3;
+    // snapshot dump sequence, start with zero
+    int32 sequence = 4;
+    // snapshot stack bytes
+    bytes stack = 5;
```

Comment:
Why use bytes? It is dangerous as a protocol. Do you really need this?

Reply:
I think it just needs storage to the backend, don't need to convert stack again. like you say, It 's dangerous, so change that.